tests/test_openapi.py

```diff
@@ -904,6 +915,15 @@ def test_example_generation(self):
 
                :param string kind:
                   Kind of resource to list.
+
+               **Example request:**
+
+               .. sourcecode:: http
+
+                  GET /resources/{kind} HTTP/1.1
+                  Host: example.com
+
```

Comment:
Any reason we have an extra line here? I assume the generation is slightly off?

Reply:
The reason is probably that there is no body. I will add a check to avoid this extra line.